wordpressorg.dev/provision/wp-config.php

```diff
@@ -83,6 +82,13 @@
 	define( 'SITE_ID_CURRENT_SITE', 1 );
 	define( 'BLOG_ID_CURRENT_SITE', 1 );
 	define( 'SUBDOMAIN_INSTALL',    false );
+	
+} elseif ( $_SERVER['HTTP_HOST'] === 'translate.wordpressorg.dev' ) {
+	define( 'GLOTPRESS_TABLE_PREFIX', 'translate_' );
```

Comment:
Or here https://github.com/WordPress/meta-environment/blob/e4a40b0a56c95c9813071315249bc80d4a2fecce/wordpressorg.dev/provision/wp-config.php#L11 which also needs to be updated.